velox/exec/PrefixSort.cpp

```diff
@@ -178,7 +208,7 @@ void PrefixSortLayout::optimizeSortKeysOrder(
       rowType->size(), std::nullopt);
   for (const auto& projection : keyColumnProjections) {
     encodedKeySizes[projection.inputChannel] = PrefixSortEncoder::encodedSize(
-        rowType->childAt(projection.inputChannel)->kind());
+        rowType->childAt(projection.inputChannel)->kind(), std::nullopt);
```

Comment:
We can pass maxStringPrefixLength from the prefix sort config from caller?

Reply:
I think we need to use a large value, such as UINT_MAX - 1, to ensure that VARCHAR columns are positioned after other supported types but before unsupported columns.